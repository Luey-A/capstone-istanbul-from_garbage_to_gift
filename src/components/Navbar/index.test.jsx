import React from "react";
import renderer from "react-test-renderer";
import SiteNavbar from "./";

it("renders ProjectDescription correctly", () => {
  const tree = renderer.create(<SiteNavbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
