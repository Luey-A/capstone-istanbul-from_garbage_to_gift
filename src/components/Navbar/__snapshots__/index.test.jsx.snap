// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProjectDescription correctly 1`] = `
<nav
  className="navMainClass navbar navbar-expand-lg navbar-light"
>
  <a
    className="navbar-brand"
    href="/"
  >
    <img
      alt="logo"
      src="../logo.png"
      width="75em"
    />
  </a>
  <button
    aria-controls="basic-navbar-nav"
    aria-label="Toggle navigation"
    className="navbar-toggler collapsed"
    onClick={[Function]}
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    aria-expanded={null}
    className="navbar-collapse collapse"
    id="basic-navbar-nav"
  >
    <div
      className="mr-auto navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        className="navMainClass nav-link"
        data-rb-event-key="/"
        href="/"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        navbarsection.Home
      </a>
      <a
        className="navMainClass nav-link"
        data-rb-event-key="/posts"
        href="/posts"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        navbarsection.Posts
      </a>
      <a
        className="navMainClass nav-link"
        data-rb-event-key="/donate"
        href="/donate"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        navbarsection.Donate
      </a>
      <a
        className="navMainClass nav-link"
        data-rb-event-key="/about"
        href="/about"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        navbarsection.AboutUs
      </a>
      <a
        className="navMainClass nav-link"
        data-rb-event-key="/contact"
        href="/contact"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        navbarsection.ContactUs
      </a>
    </div>
    <form
      className="form-inline"
    >
      <input
        className="form-control"
        placeholder="navbarsection.SearchHere"
        type="text"
      />
    </form>
    <button
      className="navButtons btn btn-primary"
      disabled={false}
      type="button"
    >
      navbarsection.Search
    </button>
    <button
      className="navButtons btn btn-primary"
      disabled={false}
      type="button"
    >
      navbarsection.LogIn
    </button>
    <button
      className="navButtons signUpButton btn btn-primary"
      disabled={false}
      type="button"
    >
      navbarsection.SignUp
    </button>
    <div
      className="navMainClass dropdown nav-item"
      onKeyDown={[Function]}
    >
      <a
        aria-expanded={false}
        aria-haspopup={true}
        className="dropdown-toggle nav-link"
        href="#"
        id="basic-nav-dropdown"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        EN
      </a>
    </div>
  </div>
</nav>
`;
